tests/auth_tests/test_models.py

```diff
@@ -155,6 +155,18 @@ def test_make_random_password(self):
         for char in password:
             self.assertIn(char, allowed_chars)
 
+    def test_create_user_no_email(self):
+        user = User.objects.create_user('user')
+        self.assertEqual(user.email, "")
+        self.assertEqual(user.username, 'user')
+        self.assertFalse(user.has_usable_password())
+
+    def test_create_super_user_no_email(self):
```

Comment:
Please move this to `auth_tests.test_basic.BasicTestCase` as `test_create_superuser_without_password_and_email()` (or similar). 

(The existing division between `test_basic` and `test_models` here isn't very clear I know... 🙂)

Reply:
The test naming convention there is shorter- `test_user_no_email` and `test_superuser`  with an explanation in the Docstring (Though they are not conventionally triple quoted!). Shall I combine with the existing superuser test, `auth_tests.test_basic.BasicTestCase.test_superuser` or have its own test?

Reply:
OK, name it as you think. 

I'd put it in it's own test, though. (I didn't check but, I think `test_superuser` is only one test because it would have been ported from the doctest in fd2b99b5f122911b895987329507ee419a159a2c)